nipype/interfaces/ants/segmentation.py

```diff
@@ -594,6 +594,86 @@ def _list_outputs(self):
         return outputs
 
 
+class antsBrainExtractionInputSpec(ANTSCommandInputSpec):
+    dimension = traits.Enum(3, 2, argstr='-d %d', usedefault=True,
+                            desc='image dimension (2 or 3)')
+    anatomical_image = File(exists=True, argstr='-a %s',
+                            desc=('Structural image, typically T1.  If more than one'
+                                  'anatomical image is specified, subsequently specified'
+                                  'images are used during the segmentation process.  However,'
+                                  'only the first image is used in the registration of priors.'
+                                  'Our suggestion would be to specify the T1 as the first image.'
+                                  'Anatomical template created using e.g. LPBA40 data set with'
+                                  'buildtemplateparallel.sh in ANTs.'),
+                            mandatory=True)
+    brain_template = File(exists=True, argstr='-e %s',
+                          desc=('Anatomical template created using e.g. LPBA40 data set with'
+                                'buildtemplateparallel.sh in ANTs.'),
+                          mandatory=True)
+    brain_probability_mask = File(exists=True, argstr='-m %s',
+                                  desc=('Brain probability mask created using e.g. LPBA40 data set which'
+                                        'have brain masks defined, and warped to anatomical template and'
+                                        'averaged resulting in a probability image.'),
+                                  copyfile=False, mandatory=True)
+    out_prefix = traits.Str('highres001_', argstr='-o %s', usedefault=True,
+                            desc=('Prefix that is prepended to all output'
+                                  ' files (default = highress001_)'))
+
+    extraction_registration_mask = File(exists=True, argstr='-f %s',
+                                        desc=('Mask (defined in the template space) used during'
+                                              ' registration for brain extraction.'
+                                              'To limit the metric computation to a specific region.'))
+    image_suffix = traits.Str('nii.gz', desc=('any of standard ITK formats,'
+                                              ' nii.gz is default'),
+                              argstr='-s %s', usedefault=True)
+    use_random_seeding = traits.Enum(0, 1, argstr='-u %d',
+                                     desc=('Use random number generated from system clock in Atropos'
+                                           '(default = 1)'))
+    keep_temporary_files = traits.Int(argstr='-k %d',
+                                      desc='Keep brain extraction/segmentation warps, etc (default = 0).')
+    use_floatingpoint_precision = traits.Enum(0, 1, argstr='-q %d',
```

Comment:
@hjmjohnson - do all ants commands now have this option, if so should this be moved to the base class?